### PR TITLE
Disable Home Offset for Delta printers

### PR DESCRIPTION
**Pull request by Guilouz**
Disabled Home Offset icon, M206 command isn’t available for Delta printers.

### CODE REVIEW

File: TFT/src/User/Menu/Tuning.c [hunk 1]
```diff
@@ -8,7 +8,11 @@ const MENUITEMS TuningItems = {
   {
     {ICON_PID,                     LABEL_PID},
     {ICON_TUNE_EXTRUDER,           LABEL_TUNE_EXTRUDER},
+    #if DELTA_PROBE_TYPE == 0  // if not Delta printer
     {ICON_PROBE_OFFSET,            LABEL_H_OFFSET},
```
Comment (digant73) [inline TFT/src/User/Menu/Tuning.c:12]: indent all the lines between #if statement by two characters

File: TFT/src/User/Menu/Tuning.c [hunk 2]
```diff
@@ -8,7 +8,11 @@ const MENUITEMS TuningItems = {
...
+    #else
+    {ICON_NULL,                    LABEL_NULL},
```
Comment (digant73) [inline TFT/src/User/Menu/Tuning.c:14]: as previous comment

File: TFT/src/User/Menu/Tuning.c [hunk 3]
```diff
@@ -37,9 +41,11 @@ void menuTuning(void)
         break;
 
       case KEY_ICON_2:
+        #if DELTA_PROBE_TYPE == 0  // if not Delta printer
         storeCmd("M206\n");
```
Comment (digant73) [inline TFT/src/User/Menu/Tuning.c:45]: as previous comment

**Comment by Foxies-CSTL** [comment]
Hi,
I also use Deltas printers with a BTT-TFT35 screen. The remark of @Guilouz is right (I also work on it) but you should not disable the "H_OFFSET" icon in the machine tuning part but rather reassign it to "P_OFFSET" with a warning message. I made a mistake when I wanted to make a merge on Master instead of a PR, sorry.
Here is the example that works for me.
27661dcab83c59968fabd8707f00a38c44e5bb2c

**Comment by Guilouz** [comment]
@Foxies-CSTL P_OFFSET is already available in ABL menu, it's not necessary to have it twice.

**Comment by Foxies-CSTL** [comment]
@Guilouz 
I know that but I found it easier to group them in the "tuning" part than to go through several menus. I do the same with the different UBL modes but I did not publish it.

**Comment by Guilouz** [comment]
But the P_Offset has no place in the Tuning menu, it's a leveling adjustment and there's no point in creating duplicates.
We don't do PID and extruder adjustment all the time, it has more place in the ABL part next to calibration and leveling.

**Pull request merged**